src/main/java/seedu/address/MainApp.java

```diff
@@ -62,36 +58,13 @@ public void init() throws Exception {
 
         initLogging(config);
 
-        model = initModelManager(storage, userPrefs);
+        model = new ModelManager(storage, userPrefs);
 
-        logic = new LogicManager(model, storage);
+        logic = new LogicManager(model);
 
         ui = new UiManager(logic);
-    }
```

Comment:
I've moved this to be under ModelManager#initModelManager method. I didnt call it new ModelManager though, because I don't want to confuse it with the constructor. initModel Manager does file and data conversion checks.

Reply:
Hmm wait so what are your plans for this? Cos im about to start writing the model intiialization code and saving code